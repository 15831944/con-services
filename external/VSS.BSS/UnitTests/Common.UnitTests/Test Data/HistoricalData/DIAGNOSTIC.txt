<?xml version="1.0" encoding="UTF-8"?>
<diagnostic:topic xmlns:diagnostic="http://www.cat.com/em/feed/v4/Diagnostic" xmlns:feed="http://www.cat.com/em/feed/v4/Common" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cat.com/em/feed/v4/Diagnostic https://emfeed.cat.com/EMFeedFiles/schema/v4/Diagnostic.xsd">
<nextBuffer>
<url></url>
<moreData>false</moreData>
</nextBuffer>

   <message>
      <header>
         <messageId>0</messageId>
         <masterMsgId>119316202</masterMsgId>
         <equipment>
            <serialNumber>-TFW00730</serialNumber>
            <make>CAT</make>
         </equipment>
         <moduleCode>-DQ0000070752Q1</moduleCode>
         <moduleTime>2009-09-16 23:39:06</moduleTime>
         <receivedTime>2009-09-17 05:03:37</receivedTime>
      </header>
      <numberOfDiagnostics>1</numberOfDiagnostics>
      <diagnosticBlock>
         <diagnostic>
            <level>2</level>
            <mid info="https://emfeed.cat.com/EMFeedService/v4/mid/81">81</mid>
            <fmi info="https://emfeed.cat.com/EMFeedService/v4/fmi/CDL/168/4">
               <cid>168</cid>
               <indicator>4</indicator>
            </fmi>
            <occurances>56</occurances>
            <timestamp>2009-09-15 04:39:57</timestamp>
         </diagnostic>
      </diagnosticBlock>
   </message>
   <message>
      <header>
         <messageId>0</messageId>
         <masterMsgId>119316953</masterMsgId>
         <equipment>
            <serialNumber>-BXY02309</serialNumber>
            <make>CAT</make>
         </equipment>
         <moduleCode>-DQ0200052993Q1</moduleCode>
         <moduleTime>2009-09-16 23:54:36</moduleTime>
         <receivedTime>2009-09-17 05:09:40</receivedTime>
      </header>
      <numberOfDiagnostics>2</numberOfDiagnostics>
      <diagnosticBlock>
         <diagnostic>
            <level>2</level>
            <mid info="https://emfeed.cat.com/EMFeedService/v4/mid/30">30</mid>
            <fmi info="https://emfeed.cat.com/EMFeedService/v4/fmi/CDL/96/3">
               <cid>96</cid>
               <indicator>3</indicator>
            </fmi>
            <occurances>102</occurances>
            <timestamp>2009-09-16 13:52:38</timestamp>
         </diagnostic>
         <diagnostic>
            <level>2</level>
            <mid info="https://emfeed.cat.com/EMFeedService/v4/mid/30">30</mid>
            <fmi info="https://emfeed.cat.com/EMFeedService/v4/fmi/CDL/96/3">
               <cid>96</cid>
               <indicator>3</indicator>
            </fmi>
            <occurances>125</occurances>
            <timestamp>2009-09-16 17:54:38</timestamp>
         </diagnostic>
      </diagnosticBlock>
   </message>
</diagnostic:topic>
