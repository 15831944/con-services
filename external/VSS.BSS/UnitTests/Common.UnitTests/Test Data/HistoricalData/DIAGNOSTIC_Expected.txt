<?xml version="1.0" encoding="utf-8"?><ArrayOfNHDataWrapper xmlns:i="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.nighthawk.com/nighthawk/service/NHData/2009/10"><NHDataWrapper><Data z:Id="i1" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataFaultDiagnostic" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>382</d3p1:AssetID><d3p1:CID>168</d3p1:CID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>Unknown</d3p1:DeviceType><d3p1:DimDatalink i:nil="true" /><d3p1:DimDatalinkReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimDatalinkReference><d3p1:DimSeverityLevel i:nil="true" /><d3p1:DimSeverityLevelReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSeverityLevelReference><d3p1:DimSource i:nil="true" /><d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSourceReference><d3p1:EventUTC>2009-09-15T04:39:57</d3p1:EventUTC><d3p1:FMI>4</d3p1:FMI><d3p1:GPSDeviceID i:nil="true" /><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:MID>81</d3p1:MID><d3p1:Occurrences>56</d3p1:Occurrences><d3p1:SourceMsgID>119316202</d3p1:SourceMsgID><d3p1:fk_DimDatalinkID>1</d3p1:fk_DimDatalinkID><d3p1:fk_DimSeverityLevelID>2</d3p1:fk_DimSeverityLevelID><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i2" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataFaultDiagnostic" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>383</d3p1:AssetID><d3p1:CID>96</d3p1:CID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>Unknown</d3p1:DeviceType><d3p1:DimDatalink i:nil="true" /><d3p1:DimDatalinkReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimDatalinkReference><d3p1:DimSeverityLevel i:nil="true" /><d3p1:DimSeverityLevelReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSeverityLevelReference><d3p1:DimSource i:nil="true" /><d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSourceReference><d3p1:EventUTC>2009-09-16T13:52:38</d3p1:EventUTC><d3p1:FMI>3</d3p1:FMI><d3p1:GPSDeviceID i:nil="true" /><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:MID>30</d3p1:MID><d3p1:Occurrences>102</d3p1:Occurrences><d3p1:SourceMsgID>119316953</d3p1:SourceMsgID><d3p1:fk_DimDatalinkID>1</d3p1:fk_DimDatalinkID><d3p1:fk_DimSeverityLevelID>2</d3p1:fk_DimSeverityLevelID><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i3" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataFaultDiagnostic" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>383</d3p1:AssetID><d3p1:CID>96</d3p1:CID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>Unknown</d3p1:DeviceType><d3p1:DimDatalink i:nil="true" /><d3p1:DimDatalinkReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimDatalinkReference><d3p1:DimSeverityLevel i:nil="true" /><d3p1:DimSeverityLevelReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSeverityLevelReference><d3p1:DimSource i:nil="true" /><d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSourceReference><d3p1:EventUTC>2009-09-16T17:54:38</d3p1:EventUTC><d3p1:FMI>3</d3p1:FMI><d3p1:GPSDeviceID i:nil="true" /><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:MID>30</d3p1:MID><d3p1:Occurrences>125</d3p1:Occurrences><d3p1:SourceMsgID>119316953</d3p1:SourceMsgID><d3p1:fk_DimDatalinkID>1</d3p1:fk_DimDatalinkID><d3p1:fk_DimSeverityLevelID>2</d3p1:fk_DimSeverityLevelID><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper></ArrayOfNHDataWrapper>